Assets/Scripts/Controllers/GameController.cs

```diff
@@ -14,7 +14,7 @@ public class GameController : MonoBehaviour
 {
     // TODO: Should this be also saved with the world data?
     // If so - beginner task!
-    public static readonly string GameVersion = "Someone_will_come_up_with_a_proper_naming_scheme_later";
+    public const string GameVersion = "Someone_will_come_up_with_a_proper_naming_scheme_later";
```

Comment:
You changed a static readonly to a constant; there are a few points for either case overall since it's in the 'main' assembly it's fine as a constant.  However, complications may occur if its in another different assembly as the main game (or any script); so basically if this (the whole localization set of scripts) has an possibility of becoming a separate assembly (like as a DLL) in the future it should stay as a static readonly else constant is better (slightly).